velero/restore/im-database-rollback.sh

```diff
@@ -0,0 +1,1513 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# set -o errexit
+set -o pipefail
+set -o errtrace
+
+# ---------- Command arguments ----------
+
+OC=oc
+YQ=yq
+NAMESPACE=""
+NUM=$#
+TEMPFILE="_TMP.yaml"
+DEBUG=0
+
+# ---------- Command variables ----------
+
+MONGO_DATA_POD=""
+# mongodb image
+ibm_mongodb_install_image="icr.io/cpopen/cpfs/ibm-mongodb-install@sha256:bfd6418bb11991586ce6e8d3152acd02f71b21bbe25428cc99de35c2f41784a2"
```

Comment:
Spoke with Adam and Yanni about this and it seems like having a prereq where we ask the user to manually mirror the 4.5 image would allow us to hardcode the image value and provide a consistent experience for anyone running into this edgecase